lib/core/src/lib/datatable/components/datatable/datatable.component.html

```diff
@@ -187,6 +189,7 @@
                 <mat-checkbox [id]="'select-file-' + idx"
                               [checked]="row.isSelected"
                               [attr.aria-checked]="row.isSelected"
+                              [class.adf-checkbox-hidden]="hoveredRowIndex !== idx && !row.isSelected"
```

Comment:
Do we need set this class based on hoveredRowIndex? I see that mat-checkbox is inside of adf-datatable-row so can't we have use :hover in scss file for row and then inside that put styles for checkbox which makes it visible?